shelley-ma/impl/src/Cardano/Ledger/ShelleyMA/Timelocks.hs

```diff
@@ -173,10 +195,8 @@ ininterval slot (SJust i_s, SJust i_f) = i_s <= slot && slot <= i_f
 -- will provide this operation. We will also need to correctly compute the witness
 -- set for this new (TxBody era) as well.
 
-type ValidityInterval = (StrictMaybe SlotNo, StrictMaybe SlotNo)
-
-txvld :: Core.TxBody era -> ValidityInterval
-txvld _ = (SNothing, SNothing)
+vldt :: Core.TxBody era -> ValidityInterval
```

Comment:
I guess this can now be defined (with appropriate `HasField` constraint) - or maybe not necessary and we can just use the field directly?

Reply:
This has been done by using a HasField constraint.